webapp/README.md

```diff
@@ -134,6 +134,25 @@ Re-factoring RSU manager to utilize Redux Toolkit for state management
    - References
      - WzdxMap.js - read and load WZDx data
 
+## Feature Flags
+
+This application has the ability to disable certain features based on environment variables. For each of these variables, the feature will be enabled if the variable is anything but 'false'. These features include:
+
+- ENABLE_RSU_FEATURES: if 'false', disable all RSU-specific features, including map, RSU data, RSU configuration, and RSU organization linking.
+- ENABLE_INTERSECTION_FEATURES: if 'false', disable all intersection-specific features, including intersection map, intersection dashboard, and intersection admin pages.
+- ENABLE_WZDX_FEATURES: if 'false', disable all wzdx-specific features, including WZDx data on the main map.
+
+These variables apply to API calls, by returning empty data if the feature is disabled.
+To aid in applying these features visually, components were created to handle the conditional rendering of these features. These components are:
+
+- <ConditionalRenderRsu>: renders children if RSU_FEATURES are enabled
+- <ConditionalRenderIntersection>: renders children if INTERSECTION_FEATURES are enabled
+- <ConditionalRenderWzdx>: renders children if WZDX_FEATURES are enabled
+- <RsuRouteGuard>: Enables routing to child compoents if RSU_FEATURES are enabled. otherwise redirects to the home page.
+- <IntersectionRouteGuard>: Enables routing to child compoents if INTERSECTION_FEATURES are enabled. otherwise redirects to the home page.

```

Comment:
```suggestion
- <IntersectionRouteGuard>: Enables routing to child components if INTERSECTION_FEATURES are enabled. otherwise redirects to the home page.
```